daemon/fqdn.go

```diff
@@ -136,7 +137,7 @@ func (d *Daemon) updateSelectorCacheFQDNs(ctx context.Context, selectors map[pol
 // dnsNameManager and DNSPoller will use the default resolver and, implicitly, the
 // default DNS cache. The proxy binds to all interfaces, and uses the
 // configured DNS proxy port (this may be 0 and so OS-assigned).
-func (d *Daemon) bootstrapFQDN(restoredEndpoints *endpointRestoreState, preCachePath string) (err error) {
+func (d *Daemon) bootstrapFQDN(possibleEndpoints map[uint16]*endpoint.Endpoint, preCachePath string) (err error) {
```

Comment:
At first the garbage collector would not find any endpoints from the endpoint manager, so it would not do anything for yet. And even after the endpoints are exposed, there should be enough slack in the GC to not drop active connections?

Reply:
I see. My main concern was whether the lack of endpoints could clear out global cache entries. But this PR is not loading into the global cache AFAICT and I'm not sure how else it could be prepopulated (maybe `--tofqdns-pre-cache` but I don't think that's often used..?)